### PR TITLE
My account create account

### PR DESCRIPTION
**Pull request by ainarssondors**
Introduced functionality allows user to register an account.

### CODE REVIEW

File: src/app/component/Form/Form.component.js [hunk 1]
```diff
@@ -118,6 +118,10 @@ class Form extends Component {
             const { current } = input;
             if (current && current.id && current.value) {
                 const { id, value } = current;
+                if (id === 'is_subscribed') {
```
Comment (alfredsgenkins) [inline src/app/component/Form/Form.component.js]: What does the specific input is doing in abstract component.

Comment (alfredsgenkins) [reply src/app/component/Form/Form.component.js]: Please check the type attribute here. Please extend this fix to radio buttons as well!

File: src/app/component/MyAccount/MyAccount.component.js [hunk 2]
```diff
@@ -68,6 +68,31 @@ class MyAccount extends Component {
...
+    onCreateAccountSuccess(fields) {
+        const { createAccount } = this.props;
+        const {
+            password,
+            email,
+            firstname,
+            lastname,
+            is_subscribed
+        } = fields;
+        const object = {
```
Comment (alfredsgenkins) [inline src/app/component/MyAccount/MyAccount.component.js]: Please give it more specific name!

Comment (alfredsgenkins) [reply src/app/component/MyAccount/MyAccount.component.js]: Notice: this object structure will change in future release.

File: src/app/component/MyAccount/MyAccount.component.js [hunk 3]
```diff
@@ -68,6 +68,31 @@ class MyAccount extends Component {
...
+            customer: {
+                firstname,
+                lastname,
+                email,
+                is_subscribed
+            },
+            password
+        };
+        createAccount(object);
```
Comment (alfredsgenkins) [inline src/app/component/MyAccount/MyAccount.component.js]: Enter here.

File: src/app/component/MyAccount/MyAccount.component.js [hunk 4]
```diff
@@ -68,6 +68,31 @@ class MyAccount extends Component {
         this.setState({ isLoading: true });
     }
 
+    onCreateAccountAttempt(fields, invalidFields) {
+        this.setState({ isLoading: !invalidFields });
+    }
```
Comment (alfredsgenkins) [inline src/app/component/MyAccount/MyAccount.component.js]: Nice solution! Please include notifications as well!

File: src/app/component/ProductSort/ProductSort.component.js [hunk 5]
```diff
@@ -68,7 +42,21 @@ class ProductSort extends Component {
     }
 
     render() {
-        const { value, sortFields, sortDirection } = this.props;
+        const {
```
Comment (alfredsgenkins) [inline src/app/component/ProductSort/ProductSort.component.js]: What this functional does?

Comment (ainarssondors) [reply src/app/component/ProductSort/ProductSort.component.js]: This is the custom select functionality. I guess my-account-frontend did not have it yet.

**Pull request merged**